_pages/en/strategy-learning-automating-improving.md

```diff
@@ -1,30 +1,56 @@
 ---
 layout: default
-title: Q1 Strategy - Continuous Learning and Improvement
+title: Q1 Strategy - Continuous Improvement and Learning
 ref: strategy-q1-2020
 lang: en
 status: posted
 sections: Ready For Use
 permalink: /strategy-learning-automating-improving.html
 ---
 
-## Q1 Strategy - Continuous Learning and Improvement
+## Q1 Strategy - Continuous Improvement and Learning
 
 The following actions apply for the 1st quarter of the 2020-2021 fiscal year (April 2020 to June 2020).
 
 Strategies and actions will be reviewed and updated quarterly based on effectiveness and current progress towards the [Medium Term IT Picture - 2025](https://sara-sabr.github.io/ITStrategy/it-picture-medium-term.html) as well as following the [Strategy Map](https://sara-sabr.github.io/ITStrategy/strategy-summary.html) (layering on the strategies).
 
-- [Actions](#actions)
 - [Goals](#goals)
+- [Actions](#actions)
+
+### Goals
+
+- Continuously improve
+- Reduce lead time for changes and delivery of services/devices
+- Recover faster incidents and errors
+- Reduce frequency of incidents and errors
+
+#### Align with GC IT Direction
+
+- [Digital Standards](https://www.canada.ca/en/government/system/digital-government/government-canada-digital-standards.html)
+- [Digital Operations Strategic Plan: 2018-2022](https://www.canada.ca/en/government/system/digital-government/digital-operations-strategic-plan-2018-2022.html)
+- [Policy](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) and [Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+  - [Mandatory Procedures for Enterprise Architecture (EA) Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602)
+  - [Mandatory Procedures on Application Programming Interfaces (API)](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32604)
+  - [Acceptable Network and Device Use](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32605)
 
 ### Actions
 
 IITB teams should:
 
-#### Use up to 20% of time to learn and improve
+#### Use up to 20% of time to improve and learn
 
 **For all the following and other related activities, use CATS codes for continuous improvement (######).**
 
+##### Improve
+
+- Understand teams performance levels and set realistic goals
+- Identify gaps in your daily work performance and establish improvement priorities
+- Gather metrics (ex.: start and end date) for things your team works on
+- Show improvement based on metrics, the [Goals](#goals) and your team's mandate
+- Review existing processes and governance structure
+- Identify issues and raise them to your manager or as Grassroots initiatives or Proof of concepts
```

Comment:
Im off network atm, but should link to grassroots inititaive and how one would launch a PoC